playbooks/installer/ospd/overcloud/run.yml

```diff
@@ -20,6 +20,7 @@
   roles:
       - {role: "installer/ospd/overcloud/storage/ceph/", when: installer.overcloud.storage.backend is defined and installer.overcloud.storage.backend == "ceph"}
       - {role: "installer/ospd/overcloud/storage/netapp/", when: installer.overcloud.storage.backend is defined and installer.overcloud.storage.backend == "netapp"}
+      - {role: "installer/ospd/loadbalancer/", when: provisioner.topology.nodes.loadbalancer is defined }
```

Comment:
instead of referencing a role twice, you can create a dependency between roles in the "meta" folder[1].
Please move the invocation there so we will have only one invocation rather than two.

[1] - http://docs.ansible.com/ansible/playbooks_roles.html#role-dependencies